### DISCUSSION TITLE
Synchronisation help

### DISCUSSION THREAD

**Discussion by dearmonty** [iterate-ch/cyberduck#14919] [Q&A]
Dear all
I do not have any technical knowledge about uploading websites. I bought Cyberduck from the Mac store. I have created a bookmark The software I use is Xway. It says that I have to "Choose Synchronize… from the File menu in Cyberduck. This will display a file dialog." However when I go to synchronise it is greyed out. Please can you tell me what I need to do to point Cyberduck at my website software. Also please could you tell mer what I need to fill in for "Path" in the book mark.
Is there a manual for Cyberduck.  Thank you for your help

**Comment by tatze96** [comment]
You can't synchronize items from within an editing application to Cyberduck directly. To synchronize or, respectively, upload the website to the server through Cyberduck, you'll have to export it from your Xway application and upload the exported files either manually or with the help of the [synchronization feature](https://docs.cyberduck.io/cyberduck/sync/) to the server afterward.

Regarding the greyed-out synchronization option:
The menu option is only activated if you have a target selected within the Cyberduck browser window.

Regarding the *Path* field:
The *Path* field allows you to connect directly to a specific folder instead of to the server root. Some servers require a default path for the connection, but you'll get that information typically provided by your server administrator or hosting provider.

So, if you want to mount a folder directly, you'll need to set the full path without shortcuts within the *Path* field or mount the server's root right-click on the desired folder and choose `New Bookmark` from the appearing context menu to create a duplicate of the current bookmark but with a set path to the selected folder.
